app/scripts/directives/createSecret.js

```diff
@@ -143,6 +143,17 @@ angular.module("openshiftConsole")
         };
 
         var linkSecretToServiceAccount = function(secret) {
+          if (!$scope.serviceAccountsNames.contains($scope.newSecret.pickedServiceAccountToLink)) {
+            var alert = {
+              name: 'createAndLink',
+              data: {
+                type: "success",
+                message: "Secret " + secret.metadata.name + " was created but couldn't be linked with service account " + $scope.newSecret.pickedServiceAccountToLink + "."
```

Comment:
@spadgett I like the idea silently fail if the user was not prompted(case of the BC/DC editor). If he was(create secret page) we can print success alert that the secret was created but also a warning the could not be linked.

@bparees we will need to have the linking action somewhere in the console eventually, but don't thing we have time to update it now, together with the TC's. Would rather stay with what we have and make it right.